ironfish-cli/src/commands/miners/mined.ts

```diff
@@ -41,17 +49,35 @@ export class MinedCommand extends IronfishCommand {
   ]
 
   async start(): Promise<void> {
-    const { args } = await this.parse(MinedCommand)
+    const { flags, args } = await this.parse(MinedCommand)
     const client = await this.sdk.connectRpc()
 
-    this.log('Scanning for mined blocks...')
+    if (flags.blockHash) {
```

Comment:
Before there was a log indicating that this command was scanning. Can we have one if a `blockHash` is provided so it's consistent?

Maybe
```ts
this.log(`Scanning for mined block for ${flags.blockHash}...`)
```